database/mssql/scripts/versions/revert/v_47_ddl_revert.sql

```diff
@@ -0,0 +1,20 @@
+SET ANSI_NULLS ON
+GO
+SET QUOTED_IDENTIFIER ON
+GO
+SET NOCOUNT ON
+GO
+
+SET XACT_ABORT ON
+GO
+SET TRANSACTION ISOLATION LEVEL SERIALIZABLE
+BEGIN TRANSACTION
+GO
+
+DELETE FROM [permit].[ORBC_PERMIT_TYPE] WHERE PERMIT_TYPE IN ('QRFR', 'STFR')
```

Comment:
Is this sufficient deletion for revert? What cascading considerations are to be made here?

Reply:
This is fine as-is. We will want to make this an outage upgrade to avoid any permits being created with those types cause then it'd get very difficult to roll back (what to do about the new permit records with those types).

Technically we shouldn't have any problems since the code to create QRFR and STFR is only in the new version and there's no reasonable way someone *could* even create one of these, but it's a safer bet to have the outage with this one. I think we may have an outage along with one of Praveen's DB changes as well so we can batch them together.

As soon as this update goes live, there will be no calling the rollback script any more - that would be used only during deployment if the app update fails for some reason after we've done the DB upgrade.

Reply:
I think this will impact /reset-modi-db.sh in lower env. The script execution will fail as there might be permits in ORBC_PERMIT